platform_plugin_communications/utils.py

```diff
@@ -0,0 +1,52 @@
+"""
+Utilities for the communications plugin.
+"""
+import hashlib
+from collections import Counter
+
+from platform_plugin_communications.edxapp_wrapper.bulk_email import get_course_email
+from platform_plugin_communications.edxapp_wrapper.instructor_tasks import (
+    InstructorTaskTypes,
+    schedule_task,
+    submit_task,
+)
+from platform_plugin_communications.tasks import send_bulk_course_email_to_learners
+
+
+def submit_bulk_course_email_to_learners(
+    request,
+    course_key,
+    email_id,
+    schedule=None,
+    extra_targets=None,
+):
+    """
+    Submit a bulk email to individual learners.
```

Comment:
Same comment here. If we're using the same function for teams/individual learners, let's update this docstring